client/src/js/modules/dc/views/samplechangerfull.js

```diff
@@ -22,7 +22,8 @@ define(['marionette',
         
         onRender: function() {
             var bl = this.getOption('bl')
-            var large = bl in app.config.pucks && app.config.pucks[bl] > 10
+            const bl_capacity = app.options.get('bl_capacity') || {}
+            var large = bl in bl_capacity && bl_capacity[bl]['pucks'] > 10
```

Comment:
```suggestion
            const large = bl in bl_capacity && bl_capacity[bl]['pucks'] > 10
```

Since we're making bl_capacity a const (can't suggest on deleted lines, so I can't make `bl` a const)